packages/demo/src/components/DataTables.vue

```diff
@@ -51,10 +51,10 @@ export default defineComponent({
         value: 'name',
         divider: true,
         width: '150px',
-        sort: false,
+        sortable: false,
```

Comment:
It seems like the second example uses `sort` not `sortable`?

Reply:
There are both variables in the interface, `sort` and `sortable`, `sort` is to indicate the initial status if needed, and `sortable ` is indicated is column sortable. Previously, only the `sort` was used in both cases, but now I have divided it into two.